stats/test/platform-routes.test.js

```diff
@@ -185,6 +185,38 @@ describe('Platform Routes HTTP request handler', () => {
       ])
     })
   })
+
+  describe('GET /wallets/daily', () => {
+    it('returns daily count of unique wallets for the given date range', async () => {
+      await givenDailyRewardTransferMetrics(pgPools.stats, '2024-01-10', [
+        { toAddress: 'to1', amount: 100, lastCheckedBlock: 1 }
```

Comment:
Can we make the field `lastCheckedBlock` optional and let the helper `givenDailyRewardTransferMetrics()` fill in a reasonable default value?